### DISCUSSION TITLE
Welcome to OneTwentyBackport-1.19.3 Discussions!

### DISCUSSION THREAD

**Discussion by FrostBreker** [FrostBreker/OneTwentyBackport-1.19.3#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CapeSword** [comment]
I confirmed the problem I messaged the other day is resolved. Stone breaks normally with minepicks.
